examples/files/basic_golang_app/main.go

```diff
@@ -0,0 +1,46 @@
+//
+// Copyright (c) nexB Inc. and others. All rights reserved.
```

Comment:
IMHO this is more than an example, but a key test file

Reply:
@pombredanne, this is a test project, to test that what we did in Go Inspector works.